### ISSUE TITLE
Report Generation

### ISSUE THREAD

**Issue by harry-bridge** [nottinghamtec/PyRIGS#348]
SU have asked for an inventory list so a PDF output for this should be added, long term plan would be to have generic report generation so if someone wanted to see just all the prices for assets or when they were purchased or all the things that need to be inspected by the rigging man that should be possible, but long term plan.

**Comment by FreneticScribbler** [comment]
Probably spreadsheet output is better than PDF...Don't have to style it for one!

**Comment by tomtom5152** [comment]
@FreneticScribbler Might I suggest CSV over excel for simplicity, although excel isn’t difficult as it’s just a ZIPed XML

**Comment by harry-bridge** [comment]
Both was the original idea, along with picking whatever fields you wanted to go into the report

**Comment by FreneticScribbler** [comment]
Indeed, but CSV will take less time to implement and therefore could be implemented earlier 😛

**Comment by harry-bridge** [comment]
Never said which you had to do first, just saying that the insurance company people once asked for a PDF so you shouldn't discount adding it because it's a bit more effort

**Comment by FreneticScribbler** [comment]
Oh they specifically asked for PDF?

**Comment by mattysmith22** [comment]
Is there any chance of getting an example report, so that I can see what needs to be on it?

**Comment by FreneticScribbler** [comment]
Weightsheet generation... 🤔

**Comment by mattysmith22** [comment]
That would be different from the inventory report, though feel free to create an issue so we can start discussion!
